src/Data/List/Relation/Unary/All/Properties.agda

```diff
@@ -388,11 +385,6 @@ concat⁻ : ∀ {xss} → All P (concat xss) → All (All P) xss
 concat⁻ {xss = []}       []  = []
 concat⁻ {xss = xs ∷ xss} pxs = ++⁻ˡ xs pxs ∷ concat⁻ (++⁻ʳ xs pxs)
 
-all⊆concat : (xss : List (List A)) → All (Sublist._⊆ concat xss) xss
-all⊆concat [] = []
-all⊆concat (xs ∷ xss) =
-  Sublist.++⁺ʳ (concat xss) Sublist.⊆-refl ∷ All.map (Sublist.++⁺ˡ xs) (all⊆concat xss)
```

Comment:
Is this a breaking change?

Reply:
It's not clear how this could be, when this lemma was only introduced last week!
I thikn that's why I was keen to do this refactoring as soon as possible, to rectify what I think  is the mistake we made by merging #2513 as a solution to #2509 without thinking more about where this lemma should live...

Reply:
Ah sorry! I didn't realise it was introduced last week! Ignore me.